NMCA_001/NMCA_001_script.lua

```diff
@@ -0,0 +1,767 @@
+    end
+
+    local LabPatrol_1 = ScenarioUtils.CreateArmyGroupAsPlatoon('UEF', 'LabPatrol_1', 'GrowthFormation')
+
+    for k, v in LabPatrol_1:GetPlatoonUnits() do
+        ScenarioFramework.GroupPatrolRoute({v}, ScenarioPlatoonAI.GetRandomPatrolRoute(ScenarioUtils.ChainToPositions('LabPatrol_1_Chain')))
+    end
+
+    local LabPatrol_2 = ScenarioUtils.CreateArmyGroupAsPlatoon('UEF', 'LabPatrol_2', 'GrowthFormation')
+
+    for k, v in LabPatrol_2:GetPlatoonUnits() do
+        ScenarioFramework.GroupPatrolRoute({v}, ScenarioPlatoonAI.GetRandomPatrolRoute(ScenarioUtils.ChainToPositions('LabPatrol_2_Chain')))
+    end
+
+    local LabPatrol_3 = ScenarioUtils.CreateArmyGroupAsPlatoon('UEF', 'LabPatrol_3', 'GrowthFormation')
+
+    for k, v in LabPatrol_3:GetPlatoonUnits() do
+        ScenarioFramework.GroupPatrolRoute({v}, ScenarioPlatoonAI.GetRandomPatrolRoute(ScenarioUtils.ChainToPositions('LabPatrol_3_Chain')))
+    end
+
+    ----------
+	-- Run AI
+	----------
+    M1OutpostAI.UEFOutpostAI()
+
+    ----------
+	-- Cinematics
+	----------
+	ForkThread(M1NIS)
+end
+
+function PlayerWin()
+    KillGame()
+end
+
+function PlayerLose(Ship)
+	----------
+	-- Player Lost Game
+	----------
+    PlayerLost = true
+
+
+    Ship:SetCanBeKilled(false)
+    Ship:SetReclaimable(false)
+
+    -- Mark objectives as failed
+    for k, v in AssignedObjectives do
+        if(v and v.Active) then
+            v:ManualResult(false)
+        end
+    end
+
+	ScenarioFramework.CDRDeathNISCamera(Ship)
+
+	local UEFUnits = GetUnitsInRect(ScenarioUtils.AreaToRect('M1'))
+    M1OutpostAI.DisableBase()
+    ScenarioFramework.EndOperationSafety()
+	
+	for k, v in UEFUnits do
+	    if v and not v:IsDead() and (v:GetAIBrain() == ArmyBrains[UEF]) then
+            v:Stop()
+	    end
+	end
+
+    local NISUnits = ScenarioUtils.CreateArmyGroup('UEF', 'PlayerLose')
+    for k, v in NISUnits do
+    	v:SetFireState('HoldFire')
+        IssueAttack({v}, Ship)
+    end
+end
+
+function KillGame()
+    ForkThread(
+        function()
+            ScenarioFramework.EndOperation(ScenarioInfo.OpComplete, ScenarioInfo.OpComplete)
+        end
+    )
+end
+
+function M1NIS()
+	local Engineer1 = ScenarioInfo.UnitNames[Player1]['Engineer_1']
+    local Engineer2 = ScenarioInfo.UnitNames[Player1]['Engineer_2']
+    local Engineer3 = ScenarioInfo.UnitNames[Player1]['Engineer_3']
+    local Engineer4 = ScenarioInfo.UnitNames[Player1]['Engineer_4']
+
+	ScenarioFramework.CreateUnitDamagedTrigger(PlayerLose, ScenarioInfo.Ship, .75)
+
+	WaitSeconds(1)
+
+	Cinematics.EnterNISMode()
+
+	ScenarioFramework.Dialogue(OpStrings.IntroNIS_Dialogue, nil, true)
+	Cinematics.CameraMoveToMarker(ScenarioUtils.GetMarker('Cutscene_01a'), 0)
+	WaitSeconds(.5)
+	Cinematics.CameraTrackEntity(Engineer1, 40, 3)
+
+	WaitSeconds(7)
+
+	Cinematics.CameraMoveToMarker(ScenarioUtils.GetMarker('Cutscene_01b'), 2)
+
+	Cinematics.ExitNISMode()
+
+    if ScenarioInfo.HumanPlayers == 2 then

```

Comment:
`ScenarioInfo.HumanPlayers` is not a number but table so this will be always false
you can use `table.getn(ScenarioInfo.HumanPlayers)` to compare to a number